pkg/models/conditionInfoConfig.go

```diff
@@ -0,0 +1,199 @@
+package models
+
+import (
+	"encoding/json"
+	"log"
+)
+
+type ConditionInfoConfig struct {
+	Name           string          `json:"name"`
+	ConditionInfos []ConditionInfo `json:"conditionInfos"`
+}
+
+type ConditionInfo struct {
+	Type     string `json:"type"`
+	Expected string `json:"expected"`
+}
+
+func defaultServingConditionConfiguration() []byte {
```

Comment:
Let’s add a comment to explain the configuration. Also, wonder if it makes sense to have this in a file?

Reply:
yes, it should be in a configuration file, but I think it is the next step to support a file & also allow user to change the file :-)